### DISCUSSION TITLE
Matek H743-Wlite frozen when taxing

### DISCUSSION THREAD

**Discussion by ghost** [iNavFlight/inav#8820] [General]
Hi, I have no idea what is happening with my FC. It gets frozen on the ground when taxing. Everything works and suddenly no response. Nothing moves. Arming status keeps blue and GPS green (15 sats). I have to unplug the battery and then it works again but after some meters the same story again. I am sending file from Blackbox. Hopefully, someone could help me, please. Many thanks.

[LOG00098.TXT](https://github.com/iNavFlight/inav/files/10806407/LOG00098.TXT)
[LOG00097.TXT](https://github.com/iNavFlight/inav/files/10806408/LOG00097.TXT)
[LOG00096.TXT](https://github.com/iNavFlight/inav/files/10806410/LOG00096.TXT)
[LOG00095.TXT](https://github.com/iNavFlight/inav/files/10806411/LOG00095.TXT)
[LOG00094.TXT](https://github.com/iNavFlight/inav/files/10806414/LOG00094.TXT)
[LOG00099.TXT](https://github.com/iNavFlight/inav/files/10806404/LOG00099.TXT)

**Comment by Jetrell** [comment]
Try turning off the blackbox logger in the configurator, or removing the SD card from the flight controller... Then try it again.
If that works. Read the stuff I wrote here - https://github.com/iNavFlight/inav/discussions/8554

**Comment by ghost** [comment]
Hi, I removed the SD card and disable the blackbox in configuratiom. I also disable the auto trim but it remains the same. It freezes when the motor is running. When I unplug the ESC it looks OK.

**Comment by Jetrell** [comment]
What PWM Protocol are you using?

**Comment by ghost** [comment]
Hi, I am on Jeti EX-bus.

**Reply by Jetrell** [reply]
Sorry. I meant the ESC protocol.. Standard, Oneshot, Multishot, Dhsot150, 300, 600.
Have you tried changing to other protocols?

**Comment by ghost** [comment]
Sorry, I did not get it. It is standard.

**Reply by Jetrell** [reply]
Is this with INAV 6.0 ? 
There was a report of a specific series of ESC's not working correctly in `Standard `at 400Hz... Which is now default, and can't be changed. When it used to be 50Hz.

**Comment by ghost** [comment]
there is no 400Hz for servos but 400kHz for I2C speed. It has nothing to do with the ESC. I ha set 50Hz for servos in iNav and in Jeti Tx, too.

**Reply by Jetrell** [reply]
>When I unplug the ESC it looks OK.

ESC's also run a specific refresh rate as well.
50Hz is the default frame rate for the servo's.. But 50Hz also used to be the default `motor_pwm_rate` for the Standard ESC PWM protocol.. Now in INAV 6.0 its set to 400Hz. And it fixed at that.

**Comment by ghost** [comment]
and yes i am on iNav 6 RC2

**Comment by ghost** [comment]
sorry, I am so over exited by all this mess around, so I do not read properly. (I crashed the plane, and do not know why) So you are saying there is 400Hz by default for standard ESC? how is is in version 5.1?

**Reply by Jetrell** [reply]
Thinking back.. The change may have been in 5.0.  
Can your ESC run Oneshot or Dshot, for a test.

**Comment by ghost** [comment]
Well, I will try it but I tried to unplug the ESC from the FC (I mean power cable) and connected it to separate battery. Now it cannot measure the current A and power W but it looks like it works. I plugged it back and the FC got frozen. I unplugged it again and plugged to another battery and it worked. I think there is something with a power board? I guess it could be somehow damaged when crashed couple days ago. Maybe, not enough power was a reason of the crash.....

**Comment by ghost** [comment]
ESC did not work with Dshot

**Comment by ghost** [comment]
Thank you very much for your time and help 👍 I am not sure if the power board was really cause, give it 99%. I will let you know after test flight.

**Comment by ghost** [comment]
I am finally more than happy. I flew 4 batteries with my Piper. It was huge victory for me after the crash. The whole problem was the power board. It was not able to handle 45A ESC. (I have to speak about it with Matek customer support) After crash, my trust to that FC was on the freezing point, so I was taxing first, then took off with a very little soul, and landed. (I discovered the FC disarm itself after landing, I like it) then took off again and did som circles, then Angle mode, Acro mode, loiter mode (changed orientation with rudder - nice feature), and rth at the and. It all worked!!! I did not do any auto tune today. :D :D :D
